ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-eus-upgrade-from-4.16.yaml

```diff
@@ -61,6 +61,16 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: aws-ipi-ovn-hypershift-replace-f28
```

Comment:
We are doing 4.20 chain upgrade jobs, any reason to change 4.18 here?

Reply:
Removed same job from chain upgrade file.